GetCanvasInbox/index.js

```diff
@@ -2,13 +2,18 @@ const axios = require("axios");
 const main = async(inputs, auths, context) => {
   console.log("Fetching data");
   let { domain, token } = inputs;
-  const {data} = await axios.get(
-    `https://${domain}/api/v1/conversations`,
-    {
-      headers: { 'Authorization': `Bearer ${token}` }
-    }
-  );
-  return data;
+  try{
```

Comment:
Do not use try catch, let Dcoder's system handle the errors. If you catch the errors, system will not know that block got an error and will try to run flow ahead. No need of try catch here.